datamodel.combodo-approval-extended.xml

```diff
@@ -87,6 +87,18 @@
           <default_value/>
           <is_null_allowed>true</is_null_allowed>
         </field>
+        <field id="target_class" xsi:type="AttributeClass">
+          <class_category>bizmodel</class_category>
+          <sql>target_class</sql>
+          <default_value>UserRequest</default_value>
+          <is_null_allowed>false</is_null_allowed>
+        </field>
+        <field id="target_class_state" xsi:type="AttributeString">
+          <sql>target_class_state</sql>
+          <class_field>target_class</class_field>
```

Comment:
```suggestion
```

What is this for? Looks like a remain of an old attempt for a different attribute type